relayer/workers/ethrelayer/worker.go

```diff
@@ -0,0 +1,144 @@
+// Copyright 2020 Snowfork
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package ethrelayer
+
+import (
+	"context"
+
+	"golang.org/x/sync/errgroup"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/snowfork/go-substrate-rpc-client/v2/types"
+	"github.com/snowfork/polkadot-ethereum/relayer/chain"
+	"github.com/snowfork/polkadot-ethereum/relayer/chain/ethereum"
+	"github.com/snowfork/polkadot-ethereum/relayer/chain/parachain"
+	"github.com/snowfork/polkadot-ethereum/relayer/crypto/secp256k1"
+	"github.com/snowfork/polkadot-ethereum/relayer/crypto/sr25519"
+)
+
+type Worker struct {
+	ethconfig  *ethereum.Config
+	ethconn    *ethereum.Connection
+	paraconfig *parachain.Config
+	paraconn   *parachain.Connection
+	log        *logrus.Entry
+}
+
+const Name = "EthereumRelayer"
```

Comment:
We should load this from config as I suggested in https://github.com/Snowfork/polkadot-ethereum/pull/354/files#r617686493

Also a naming guideline, a name like `relay:ethereum➞parachain` would be more descriptive.

Reply:
i'm gonna migrate this worker in a new PR, that keeps to existing conventions. should have PR up in an hour or so

then in a future PR we can migrate all workers to use new conventions around logging, metrics, config, etc